music-release.json

```diff
@@ -0,0 +1,8 @@
+{
+    "type": "music-release",
+    "title": "(miniatura070) bleupulp - the fall of",
+    "archivedotorg": "https://archive.org/details/Miniatura070Bleupulp-theFallOf",
+    "license":"https://creativecommons.org/licenses/by-nc-nd/3.0/",
+    "cover": "&2JGx5HkFts0PS84K3xwKGZZgqQ5zAf2LSuLAahBhm6I=.sha256",
+    "ipfs": "/ipfs/QmSuUXxGsF7yQNPDQEjUsAmWZZoNNmdMmZK59MsQroX3ch"
+}
```

Comment:
Oh.. yea, sorry about that.
I could have documented these inside the rendering js files as comments.